nixos/modules/services/torrent/qbittorrent-nox.md

```diff
@@ -0,0 +1,15 @@
+# qBittorrent-nox {#module-services-torrent-qbittorrent-nox}
+
+qBittorrent-nox is available as a standalone package as `qbittorrent-nox`, it can be installed to test the program before setting up the service.
+
+Options can be decleared like the following. On first start up check the logs with `journelctl -u qbittorrent-nox`, You'll need the temporary password to login to the webui as the admin account, Then you can go to the gear icon -> `Web UI` -> `Authentication` and change the password. [You can connect to the webui at port 8080 as shown in the example below.](http://localhost:8080/)
+```nix
+services.qbittorrent-nox = {
+  enable = true;
+
+  port = 8080;
+  openFirewall = true;
+  user = "root";
```

Comment:
Since this will likely be copy-pasted, it's better to either remove this line or replace it with a specific username.

Reply:
I disagree, I added it here because I had problems with sonarr being unable to have permissions to hardlink files because it did not have rw permissions to qbittorrent's downloads folder. For sure I have a skill issue with linux permissions but having everything under root makes it easier at first.

Reply:
Running things as root must be discouraged as much as possible, especially in the official documentation.